.gitignore

```diff
@@ -39,3 +39,6 @@ install_manifest.txt
 # ProjectFiles
 *.pro.user*
 *.user
+
+# Package manager
+vcpkg_installed
```

Comment:
This may be unnecessary because `vcpkg_installed` is created in `build` directory by default.  
(Executing build command on the root directory is not recommended.)

Reply:
**My makefile is gitignored, whoops.
Why is executing build on root not recommended?
The build output will still be generated in the "build" folder.**

For me I run - on macos:

**1. Install dependency - in this case `sndfile` according to `vcpkg.json`**
```
vcpkg install --triplet wasm32-emscripten
```
This creates a `vcpkg_installed` folder in project root.

**2. Build file**
```
emcmake cmake -B build/wasm-libsndfile --preset wasm-libsndfile
emmake make -C build/wasm-libsndfile
```

**Here is a the makefile I used that is gitignored:**
```
install:
	vcpkg install --triplet wasm32-emscripten

wasm:
	emcmake cmake -B build/wasm-vanilla --preset wasm
	emmake make -C build/wasm-vanilla

wasm-libsndfile:
	emcmake cmake -B build/wasm-libsndfile --preset wasm-libsndfile
	emmake make -C build/wasm-libsndfile

clean:
	rm -rf build
```
Preset options are written in `CMakePresets.json`
Then we can add different installation scripts to makefile.

**Build instructions**
```
make install
make wasm-libsndfile
```